test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -430,12 +430,24 @@ def get_system_info(self, hostname=None):
             _msg = 'failed to get content of /proc/meminfo of host: '
             self.logger.error(_msg + hostname)
         else:
+            got_MemAvailable = False
             for i in mem_info['out']:
                 if "MemTotal" in i:
                     self.system_total_ram = float(i.split()[1]) / (2**20)
                 elif "MemAvailable" in i:
-                    self.system_ram = float(i.split()[1]) / (2**20)
+                    MemAvailable = float(i.split()[1]) / (2**20)
+                    got_MemAvailable = True
```

Comment:
just checked the pep8 standard, variable names should be all lower case, so let us use got_mem_available... apologies for not getting it right the first time